tests/integration/targets/aws_api_gateway_domain/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/aws
+aws_api_gateway_domain
```

Comment:
Not sure what the conventions for aliases are in this new community.aws project yet.

Reply:
Aliases/conventions are the same in the aws collections as they were in ansible/ansible